src/main/webapp/resources/js/pages/sequence-files/components/FastQCCharts.jsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * This file renders the FastQC charts component.
+ */
+
+import React, { useEffect, useState } from "react";
+import { Col, Layout , Row, Typography} from "antd";
+import { SPACE_MD } from "../../../styles/spacing";
+import { grey1 } from "../../../styles/colors";
+import { TabPaneContent } from "../../../components/tabs/TabPaneContent";
+import { getFastQCImages } from "../../../apis/files/sequence-files";
+import { ContentLoading } from "../../../components/loader";
+import {
+  seqObjId,
+  seqFileId
+} from "../fastqc-constants";
+
+import styled from "styled-components";
+
+const StyledImage = styled.img`
+  height: 100%;
+  width: 100%;
+`;
+
+export default function FastQCCharts() {
+
+  const [loading, setLoading] = useState(true);
+  const [perBase, setPerBase] = useState("");
+  const [perSeq, setPerSeq] = useState("");
+  const [duplicationLevel, setDuplicationLevel] = useState("");
+  const [fastQCVersion, setFastQCVersion] = useState("");
+
+  useEffect(() => {
+    getFastQCImages(seqObjId, seqFileId).then(({ perbaseChart, persequenceChart, duplicationlevelChart, fastQCVersion }) => {
+      // Convert the images from byte arrays into a png images
+      setPerBase(`data:image/png;base64,${perbaseChart}`);
+      setPerSeq(`data:image/png;base64,${persequenceChart}`);
+      setDuplicationLevel(`data:image/png;base64,${duplicationlevelChart}`);
+
+      setFastQCVersion(fastQCVersion);
+      setLoading(false);
+    });
+  }, []);
+
+  return (
+    <div>
+    {
+      loading ?
+        <ContentLoading message={i18n("FastQC.fetchingCharts")}/>
+        :
+        <Layout style={{paddingLeft: SPACE_MD, backgroundColor: grey1}}>
+          <TabPaneContent title={i18n("FastQC.charts")} xxl={16}>
+            <Typography.Paragraph>{i18n("FastQC.overrepresentedSequencesDescription", fastQCVersion)}</Typography.Paragraph>
+            <Row
+              gutter={[16, 16]}
+              style={{ padding: SPACE_MD }}
+            >
+              <Col sm={24} md={18} xl={16} xxl={12}>
```

Comment:
Images look a bit small now.  You may want to consider increasing the widths on the Col spans a little.  Playing a bit just with `<Col span={24}>` seemed to work ok.  On wider screens it gets really big so maybe set an xl or xxl breakpoint, but I think I would skip the smaller ones.

Reply:
Updated in 139845ae0399cc73ea9aa42936a8341d24590480